configs/default/KLEE-SYNERGYF4.config

```diff
@@ -127,7 +127,7 @@ set pid_process_denom = 1
 set system_hse_mhz = 8
 set max7456_spi_bus = 3
 set dashboard_i2c_bus = 2
-set pinio_box = 39,255,255,255
+set pinio_box = 40,255,255,255
```

Comment:
@lexiotech: This will require the documentation that you provide for this board to be updated. Can you please confirm that you'll be doing this?